src/commonMain/kotlin/com/configcat/ConfigCatClient.kt

```diff
@@ -289,19 +289,19 @@ internal class Client private constructor(
         val result = getSettings()
         val evalUser = user ?: defaultUser
         if (result.settings.isEmpty()) {
-            val message = "Config JSON is not present. Returning defaultValue: '$defaultValue'."
+            val message = "Config JSON is not present. Returning the `defaultValue` parameter that you specified in your application: '$defaultValue'."
             val details = EvaluationDetails.makeError(key, defaultValue, message, evalUser)
-            logger.error(message)
+            logger.error("[1000] " + message)
             hooks.invokeOnFlagEvaluated(details)
             return details
         }
 
         val setting = result.settings[key]
         if (setting == null) {
-            val message = "Value not found for key '$key'. Here are the available keys: " +
+            val message = "Failed to evaluate setting '$key' (the key was not found in config JSON). Returning the `defaultValue` parameter that you specified in your application: '$defaultValue'. Available keys: " +
```

Comment:
Available keys should be surrounded with single quotes like 'Key1', 'Key2', 'Key3'